src/engine/enginebuffer.cpp

```diff
@@ -627,10 +625,12 @@ bool EngineBuffer::updateIndicatorsAndModifyPlay(bool newPlay, bool oldPlay) {
     // allow the set since it might apply to a track we are loading due to the
     // asynchrony.
     bool playPossible = true;
+    struct QueuedSeek queuedSeek = m_queuedSeek.getValue();
```

Comment:
The extra struct/enum keyword are not needed in C++ (multiple occurrences). Instead better use const here.